Serializable/Serializable/Classes/Serializable.swift

```diff
@@ -25,7 +25,7 @@ public protocol Decodable {
 }
 
 public extension Decodable {
-    public static func array(_ source: AnyObject?) -> [Self] {
+    public static func array(_ source: Any?) -> [Self] {
```

Comment:
This was changed because NSDictionary's object(forKey:) method returns Any now and not AnyObject.